pkg/driver/driver.go

```diff
@@ -53,7 +53,8 @@ const (
 
 	WellKnownTopologyKey = "topology.kubernetes.io/zone"
 	// DEPRECATED Use the WellKnownTopologyKey instead
-	TopologyKey = "topology." + DriverName + "/zone"
+	TopologyKey   = "topology." + DriverName + "/zone"
+	OSTopologyKey = "kubernetes.io/os"
```

Comment:
Thoughts on considering "topology." + DriverName + "/os" as an alternative?

Reply:
`kubernetes.io` is a [well-known label](https://kubernetes.io/docs/reference/labels-annotations-taints/#kubernetes-io-os), we should use it - if we wanted we could _also_ specify another label, but I don't think that's necessary